### ISSUE TITLE
The CEP2D subroutine diffusion tensor - spatial isotropy computation is wrong

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svFSI#109]
While converting Fortran code to C++ I see the following code in `SUBROUTINE CEP2D`

```
!        Diffusion tensor - spatial isotropy
         Diso    = Diso * Jac
         Dani(:) = Dani(:) * Jac
         D(:,:)  = Diso * C(:,:)
```

I believe the `D(:,:)  = Diso * C(:,:)` statement should be `D(:,:)  = Diso * Ci(:,:)`.

**Comment by vvedula22** [comment]
@ktbolt thanks.. yes, this is a bug.. will fix it in the next commit

**Comment by vvedula22** [comment]
Solved in pull request #110
